### PR TITLE
Remove postRouteServerMiddleware

### PR DESCRIPTION
**Pull request by nramadas**
:eyeglasses: @phil303 @schwers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # r/platform
 A set of tools to enable easy universal rendering and page navigation on a React + Redux stack.
 
+## Change Log
```
Comment (schwers-zz) [inline README.md:4]: :100:

Comment (phil303) [reply README.md:4]: 💯

**Comment by phil303** [comment]
:fish:

**Comment by schwers-zz** [comment]
Overall patch looks good.

Can you add some more info about why we did this to the body of the commit message? It would be nice to not run into the same commit log vagueity we had in tracking this down. https://medium.com/brigade-engineering/the-secrets-to-great-commit-messages-106fc0a92a25#.91r1knm9e

**Comment by schwers-zz** [comment]
💅

**Comment by nramadas** [comment]
:haircut:

**Comment by schwers-zz** [comment]
:fish:

**Pull request merged**